### PR TITLE
Fix ::FakeFS::IO.binread forwarding to ::FakeFs::File.binread

### PR DESCRIPTION
**Pull request by Annih**
Oops.

### CODE REVIEW

**Review by grosser — COMMENTED** [review]
needs a test then ?

File: README.md [hunk 1]
```diff
@@ -86,6 +86,11 @@ require 'fakefs/safe'
 FakeFS.activate!(io_mocks: true)
 # your code
 FakeFS.deactivate!
+
+#or
+FakeFS(io_mocks: true) do
+  # your code
+end
```
Comment (grosser) [inline README.md:93]: feels a bit overkill given that almost nobody will use this ...

Comment (Annih) [reply README.md]: OK I removed my use of #478

**Comment by grosser** [comment]
1.8.0

**Pull request merged**